lerna.json

```diff
@@ -5,9 +5,6 @@
     "example",
     "example/lambdas/*",
     "example/scripts/*",
-    "lambdas/db-migration",
-    "lambdas/db-provision-user-database",
-    "lambdas/sqs-message-remover",
```

Comment:
What does this lerna.json do? We still have these lambdas referenced here. I think I just don't understand what this is doing.

Reply:
So these lines are part of the 'packages' configuration object (array) that tell lerna what the list of monorepo packages are that should be included when lifecycle scripts are run.  

In this case, we're removing the specific package directories and setting it to look for lambda/*

Reply:
Cool, totally makes sense